utils/make_rttm.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Hitachi, Ltd. (Yusuke Fujita)
+#           2021 Johns Hopkins University (Jiatong Shi)
+# Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import argparse
+from espnet2.fileio.npy_scp import NpyScpReader
+import logging
+import numpy as np
+from scipy.signal import medfilt
+
```

Comment:
Probably, the CI error comes from this file.

We automatically generate the usage of command line tools in our documentation, and to generate it,  the script need to have get_parser() function. e.g. https://github.com/espnet/espnet/blob/master/utils/compute-fbank-feats.py

\ I don't know detail.